src/main/java/seedu/foodrem/commons/util/ValidationUtil.java

```diff
@@ -0,0 +1,67 @@
+package seedu.foodrem.commons.util;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.time.format.ResolverStyle;
+
+/**
+ * Helper functions for handling validation.
+ */
+public class ValidationUtil {
+
+    /**
+     * Returns true if a string can be parsed into a double, false otherwise.
+     *
+     * @param string a string to be parsed.
+     */
+    public static boolean isParsableDouble(String string) {
+        try {
+            Double.parseDouble(string);
```

Comment:
Let us make the parameter name more clear.

```suggestion
     * @param doubleString a string to be parsed.
     */
    public static boolean isParsableDouble(String doubleString) {
        try {
            Double.parseDouble(doubleString);
```

Reply:
Thanks for the suggestion!